src/components/ParentCreatedPage.tsx

```diff
@@ -0,0 +1,129 @@
+    totalPages,
+    formPages,
+}): JSX.Element => {
+    const progressBarIncrement = Math.ceil(100 / totalPages);
+    const getProgressBarValue = (pageNum) =>
+        progressBarIncrement * (pageNum + 1);
+
+    const formPageHeaders = [
+        "Participant Information",
+        "Participant Information",
+        "Participant Emergency Form",
+        "Participant Health Form",
+        "Parent Guardian Information",
+        "Proof of Income",
+        "How did you hear about us?",
+    ];
+    return (
+        <Wrapper session={session}>
+            {pageNum < totalPages ? (
+                <Center>
+                    <Box w={912}>
+                        <Flex
+                            alignItems={"center"}
+                            justifyContent={"space-between"}
+                        >
+                            <BackButton
+                                onClick={
+                                    pageNum > 0
+                                        ? () =>
+                                              setPageNum((prevPage) =>
+                                                  Math.max(prevPage - 1, 0),
+                                              )
+                                        : null
+                                }
+                            />
+                            <CloseButton />
+                        </Flex>
+                        <Text fontWeight="700" fontSize="36px">
+                            {formPageHeaders[pageNum]}
+                        </Text>
+                        <Stack spacing={8}>
+                            <Progress
+                                value={getProgressBarValue(pageNum)}
+                                size="sm"
+                                color={colourTheme.colors.Blue}
+                                mt={8}
+                                mb={6}
+                            />
+                            {formPages.map((formPage, idx) => {
+                                return (
+                                    <Box
+                                        key={idx}
+                                        display={
+                                            pageNum === idx ? null : "none"
+                                        }
+                                    >
+                                        {formPage}
+                                    </Box>
+                                );
+                            })}
+                        </Stack>
+                    </Box>
+                </Center>
+            ) : (
+                <Center>
+                    <VStack mt={120} mb={180} spacing={50}>
+                        <Text fontWeight="700" fontSize="24px" align="center">
+                            Account created successfully
+                        </Text>
+                        <Text maxW={512}>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing
```

Comment:
Should the text here be changed?

Reply:
^^ Facts. @brandonwong368 we can replace the text with `Your account has been successfully created. Click the button below to start browsing classes!` as per the Figma